forms-flow-web/src/components/Form/Create.js

```diff
@@ -221,13 +224,20 @@ const Create = React.memo(() => {
                 <Translation>{(t) => t("Title")}</Translation>
               </label>
               <input
+                style={{ color: `${!isValidName ? "red" : ''}` }}
                 type="text"
                 className="form-control"
                 id="title"
                 placeholder={t("Enter the form title")}
                 value={form.title || ""}
-                onChange={(event) => handleChange("title", event)}
+                onChange={(event) => {
+                  setIsValidName(searchValidator(event.target.value));
```

Comment:
Could you please changethe name of the function searchValidator to inputValidator

Reply:
sure